### DISCUSSION TITLE
Brizian Disk test

### DISCUSSION THREAD

**Discussion by IntePNum** [peridigm/peridigm#339] [General]
Hey, professors,

I calculated a Brizian Disk,

Why the damage doesn't initiate begain from its center

![image](https://github.com/peridigm/peridigm/assets/108223279/733c76c6-c042-4f72-b48d-62a7ec321edc)

**Comment by oldninja** [comment]
Hi! I've experienced the same issue in my models, either with Peridigm or my own Peridynamic code. I am not entirely sure why this happens, but the way I overcame the issue was by assigning no damage to the surface of contact.
